pkg/tasktesting/run.go

```diff
@@ -33,6 +33,7 @@ type TestCase struct {
 	TaskVariant string
 	// Map workspace name of task to local directory under test/testdata/workspaces.
 	WorkspaceDirMapping map[string]string
+	TaskParamsMapping   map[string]string
```

Comment:
Can you explain a bit why did you introduce this? It looks to me like we already have the ability to set params?

Reply:
correct, it was all ready but just missing the bridge to testcases, see https://github.com/opendevstack/ods-pipeline/pull/449/files#diff-e93a7ccef5afc540dce5bccf53f1ff077858007e0a160ec249479bd7401a97eaR75 and https://github.com/opendevstack/ods-pipeline/blob/3b85ab8190f5ea9cc151df1be2304ab33437b073/pkg/tasktesting/taskrun.go#L26